### ISSUE TITLE
Prerequisites for installation on OpenShift lacking details

### ISSUE THREAD

**Issue by kevprice83** [Kuadrant/docs.kuadrant.io#133]
The [installation guide](https://docs.kuadrant.io/0.8.0/kuadrant-operator/doc/install/install-openshift/) mentions the following in the prerequisites:

>AWS account with Route 53 and zone.

There is in fact a lot of work to be done to satisfy this requirement that I think could be facilitated by some additional details on why Kuadrant needs this, what is the purpose of the various records and the roles & responsibilities of the components within Kuadrant regarding this external service and its configuration.

I have documented the steps required to perform a successful integration with Route 53 [here](https://api-mgmt.pages.redhat.com/team-docs/knowledge_sharing_sessions/knowledge_session19/#prerequisites). I wouldn't expect such detailed steps to be documented for Kuadrant but maybe we can link to the [AWS documentation](https://repost.aws/knowledge-center/create-subdomain-route-53) that I had to follow to be able to do this and explain the relationship between the various configuration and records to be created with the different Kuadrant components responsible for managing this?

As I attempted to do in the guide I wrote I think it's important for the user to understand the roles of the following resources:

- `ManagedZone`
- `HTTPRoute`
- `DNSPolicy`

Because to someone unfamiliar with Kuadrant it is not obvious and knowing this can also help the user self-resolve any issues that occur during the process rather than figuring it out themselves and ultimately opening a ticket before they have even completed the prerequisites successfully.

Let me know what you think and whether any of the guide I have written makes sense in the upstream documentation and I would be happy to submit a PR.

**Comment by jayachristina** [comment]
there is some content documented here that could help with the prerequisites

https://developers.redhat.com/articles/2024/06/12/getting-started-red-hat-connectivity-link-openshift#install_connectivity_link

**Comment by smccarthy-ie** [comment]
@kevprice83 This is great feedback, thanks. Yes, I think we might need to add links to cloud DNS provider docs. 
 
Note that the requirement to create a `ManagedZone` will no longer apply in v0.9. See https://github.com/Kuadrant/kuadrant-operator/pull/793

**Comment by kevprice83** [comment]
@jayachristina yup the [document I linked](https://api-mgmt.pages.redhat.com/team-docs/knowledge_sharing_sessions/knowledge_session19/#prerequisites) essentially describes in detail those steps. However, I would not expect those to be in the official guide but maybe some links and any appropriate context/guidance to accompany that which is probably what @smccarthy-ie is suggesting as well.

@smccarthy-ie I'm about to try installing latest so will come back to this issue in case something significant changes regarding my original proposal in this issue.

**Comment by david-martin** [comment]
Unfortunately the original link gives a 404 https://docs.kuadrant.io/0.8.0/kuadrant-operator/doc/install/install-openshift/
but that version of the doc is still available for 1.0.x https://docs.kuadrant.io/1.0.x/kuadrant-operator/doc/install/install-openshift/

It looks like the wording in the prerequisites has changed to "AWS/Azure or GCP with DNS capabilities"
It sounds reasonable to have links out to https://repost.aws/knowledge-center/create-subdomain-route-53 and equivalent Azure and GCP documentation, if available. 
At the very least, link to the landing pages for those services.
- https://learn.microsoft.com/en-us/azure/dns/dns-overview
- https://cloud.google.com/dns

Note that the document has been rewritten on 'main' to be less openshift specific (how to install with OLM),
and the url is different https://docs.kuadrant.io/dev/install-olm/
but the above still applies to the prerequisites section
